projects/storefrontlib/src/cms-components/myaccount/order/replenishment-order-history/replenishment-order-history.component.html

```diff
@@ -0,0 +1,223 @@
+<pre>
+  <!-- {{ replenishmentOrders$ | async | json }} -->
+</pre>
+
+<ng-container *ngIf="replenishmentOrders$ | async as replenishmentOrders">
+  <div class="container">
+    <!-- HEADER -->
+    <div class="cx-order-history-header">
+      <h3>
+        {{ 'orderHistory.replenishmentOrderHistory' | cxTranslate }}
+      </h3>
+    </div>
+
+    <!-- BODY -->
+    <div class="cx-order-history-body">
+      <ng-container
+        *ngIf="replenishmentOrders.pagination.totalResults > 0; else noOrder"
+      >
+        <!-- Select Form and Pagination Top -->
+        <div class="cx-order-history-sort top row">
+          <div
+            class="cx-order-history-form-group form-group col-sm-12 col-md-4 col-lg-4"
+          >
+            <cx-sorting
+              [sortOptions]="replenishmentOrders.sorts"
+              [sortLabels]="getSortLabels() | async"
+              (sortListEvent)="changeSortCode($event)"
+              [selectedOption]="replenishmentOrders.pagination.sort"
+              placeholder="{{ 'orderHistory.sortByMostRecent' | cxTranslate }}"
+            ></cx-sorting>
+          </div>
+          <div class="cx-order-history-pagination">
```

Comment:
add a condition to only show when the total pages is bigger than 1 because it doesn't make sense to display the pagination when there's only 1 page

Reply:
I saw the change was made to the child of where I said to put it at.

Put it in the div please